recipe/activate-binutils.ps1

```diff
@@ -0,0 +1,5 @@
+$Env:CONDA_BACKUP_AR=$Env:AR
+$Env:AR="@CHOST@-ar.exe"
+
+$Env:CONDA_BACKUP_RANLIB=$Env:RANLIB
+$Env:RANLIB="@CHOST@-ranlib.exe"
```

Comment:
binutils activation scripts are used in binutils-feedstock now.

Reply:
There are still .bat and .sh files for binutils in this repo -- are those just legacy?

Reply:
Yeah, they should be deleted

Reply:
Removed

Reply:
(I'm happy to delete the .sh and .bat if you want, too, but didn't touch those for now)

Reply:
Let's delete those and `install-binutils.sh` too

Reply:
Done!

Reply:
Binutils PR opened: https://github.com/conda-forge/binutils-feedstock/pull/87